### PR TITLE
Updating weather on site

### PR DESCRIPTION
**Pull request by mareliefi**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+$(function () { // Same as document.addEventListener("DOMContentLoaded"...
```
Comment (HAASLEWER) [inline js/script.js]: Remove comment or make it relevant.

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  $.get("http://api.openweathermap.org/data/2.5/weather?q=Pretoria,za&units=metric&APPID=ad072cb9c0700a66a327e65573d6cee8", function(response) {
+    var weatherresult = response;
```
Comment (HAASLEWER) [inline js/script.js]: Please use cameCase.

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    var name = "City: " + weatherresult.name;
+    var temp = "Temperature: " + weatherresult.main.temp + "°C";
+    var tempMax = "Maxmimum Temperature: " + weatherresult.main.temp_max + "°C";
+    var tempMin = "Minimum Temperature: " + weatherresult.main.temp_min + "°C";
+    var pressure = "Pressure: " + weatherresult.main.pressure + " Pa";
+    var humidity = "Humidity: " + weatherresult.main.humidity + " g/m3";
+    var weatherdescription = "Weather description: " + weatherresult.weather[0].main;
+    console.log(weatherresult);
```
Comment (HAASLEWER) [inline js/script.js]: Remove print.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+$(function () { // Same as document.addEventListener("DOMContentLoaded"...
+  $.get("http://api.openweathermap.org/data/2.5/weather?q=Pretoria,za&units=metric&APPID=ad072cb9c0700a66a327e65573d6cee8", function(response) {
+    var weatherresult = response;
+    var name = "City: " + weatherresult.name;
+    var temp = "Temperature: " + weatherresult.main.temp + "°C";
+    var tempMax = "Maxmimum Temperature: " + weatherresult.main.temp_max + "°C";
+    var tempMin = "Minimum Temperature: " + weatherresult.main.temp_min + "°C";
+    var pressure = "Pressure: " + weatherresult.main.pressure + " Pa";
+    var humidity = "Humidity: " + weatherresult.main.humidity + " g/m3";
+    var weatherdescription = "Weather description: " + weatherresult.weather[0].main;
```
Comment (HAASLEWER) [inline js/script.js]: What happens if weatherresult.weather array is empty?

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+$(function () { // Same as document.addEventListener("DOMContentLoaded"...
+  $.get("http://api.openweathermap.org/data/2.5/weather?q=Pretoria,za&units=metric&APPID=ad072cb9c0700a66a327e65573d6cee8", function(response) {
```
Comment (HAASLEWER) [inline js/script.js:2]: What happens if there's an error?  https://en.wikipedia.org/wiki/Defensive_programming

File: js/script.js (hunk 2 above)
Comment (HAASLEWER) [inline js/script.js]: What happens if the response is empty?

**Review by HAASLEWER — CHANGES_REQUESTED** [review]

File: js/script.js (hunk 1 above)
Comment (mareliefi) [reply js/script.js]: No problem dude

File: js/script.js (hunk 2 above)
Comment (mareliefi) [reply js/script.js]: Done and dusted!

File: js/script.js (hunk 3 above)
Comment (mareliefi) [reply js/script.js]: Oops I did it again

File: js/script.js (hunk 4 above)
Comment (mareliefi) [reply js/script.js]: .... only knows

File: js/script.js (hunk 5 above)
Comment (mareliefi) [reply js/script.js:2]: BaZinga!

File: js/script.js (hunk 2 above)
Comment (mareliefi) [reply js/script.js]: Response = its empty silly :)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Signika&display=swap" rel="stylesheet">
+    <title>Pretoria Weather Checker</title>
+  </head>
+<body>
+    <div class="weather-box"><h1 id="weatherHeading"></h1>
```
Comment (HAASLEWER) [inline index.html]: Move H1 to next line.

**Review by HAASLEWER — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (mareliefi) [reply index.html]: Donesies:)

**Review by HAASLEWER — APPROVED** [review]

**Pull request merged**